playbooks/scenario45_dms_inst.yaml

```diff
@@ -0,0 +1,114 @@
+---
+
+# Scenario 45: DMS
+#
+- name: Scenario 45 - DMS Kafka Instance
+  hosts: localhost
+  vars:
+    prefix: scenario45-
+  tasks:
+    - set_fact:
+        prefix: "{{ (prefix + ( lookup('env', 'TASK_EXECUTOR_JOB_ID') | default(99999999 | random | to_uuid | hash('md5'), true) ) )[0:24] }}"
+
+    - set_fact:
+        test_security_group_name: "{{ (prefix + '-sg') }}"
+        test_vpc_name: "{{ ( prefix + '-vpc') }}"
+        test_subnet_name: "{{ ( prefix + '-subnet') }}"
+        dms_inst_name: "{{ (prefix + '-dms_apimon') }}"
+        dms_spec: dms.instance.kafka.cluster.c3.mini
+
+    - block:
+      - name: List DMS az
+        script: "list_dms_az.py"
+        args:
+          executable: python3
+        register: azs
+
+      - name: Get AZ name
+        set_fact:
+          az_id: "{{ azs.stdout_lines | random }}"
+
+      - name: Get DMS product spec
+        script: "dms_list_products.py {{ dms_spec }}"
+        args:
+          executable: python3
+        register: spec
+
+      - name: Create VPC
+        opentelekomcloud.cloud.vpc:
+          name: "{{ test_vpc_name }}"
+          cidr: "10.10.0.0/24"
+          state: present
+        register: newvpc
+
+      - name: Create subnet for VPC
+        opentelekomcloud.cloud.subnet:
+          name: "{{ test_subnet_name }}"
+          vpc_id: "{{ newvpc.vpc.id }}"
+          cidr: "10.10.0.0/27"
+          gateway_ip: "10.10.0.1"
+          dns_list:
+            - "100.125.4.25"
```

Comment:
please check whether this is mandatory parameter.
If not you can remove dns_list at all